internal/identity/google.go

```diff
@@ -124,7 +124,7 @@ func (p *GoogleProvider) Revoke(accessToken string) error {
 // prompt=consent to the OAuth redirect URL and will always return a refresh_token.
 // https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess
 func (p *GoogleProvider) GetSignInURL(state string) string {
-	return p.oauth.AuthCodeURL(state, oauth2.AccessTypeOffline, oauth2.ApprovalForce)
+	return p.oauth.AuthCodeURL(state, oauth2.AccessTypeOffline, oauth2.SetAuthURLParam("prompt", "select_account consent"))
```

Comment:
@desimone according to https://stackoverflow.com/questions/23436617/force-google-to-show-account-chooser-and-consent-screen, it should be `select_account+consent` (notice the `+` sign)

Reply:
@desimone indeed